ale_linters/php/phan.vim

```diff
@@ -1,36 +1,71 @@
-" Author: diegoholiveira <https://github.com/diegoholiveira>
+" Author: diegoholiveira <https://github.com/diegoholiveira>, haginaga <https://github.com/haginaga>
 " Description: static analyzer for PHP
 
 " Define the minimum severity
 let g:ale_php_phan_minimum_severity = get(g:, 'ale_php_phan_minimum_severity', 0)
 
+let g:ale_php_phan_executable = get(g:, 'ale_php_phan_executable', 'phan')
+let g:ale_php_phan_use_client = get(g:, 'ale_php_phan_use_client', 0)
+
+function! ale_linters#php#phan#GetExecutable(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'php_phan_executable')
+    if g:ale_php_phan_use_client == 1 && l:executable is# 'phan'
```

Comment:
Put a new line before if statements that don't start a new level of indentation.